babel.config.json

```diff
@@ -49,12 +49,7 @@
     },
     "browser": {
       "presets": [
-        [
-          "@babel/preset-env",
-          {
-            "debug": false
-          }
-        ]
+        "@babel/preset-env"
```

Comment:
Can we please return back the extended syntax here? We'll have to rewrite it in future if we want to turn the debugging on.